src/Whip_WPDismissOption.php

```diff
@@ -11,23 +11,23 @@ class Whip_WPDismissOption implements Whip_DismissStorage {
 	/**
 	 * Saves the value to the options.
 	 *
-	 * @param string $dismissedVersion The value to save.
+	 * @param string $dismissedValue The value to save.
 	 *
 	 * @return bool True when successful.
 	 */
-	public function set( $dismissedVersion ) {
-		return update_option( $this->optionName, $dismissedVersion );
+	public function set( $dismissedValue ) {
+		return update_option( $this->optionName, $dismissedValue );
 	}
 
 	/**
 	 * Returns the value of the whip_dismissed option.
 	 *
-	 * @return string Returns the value of the option or an empty string when not set.
+	 * @return string|int Returns the value of the option or an empty string when not set.
```

Comment:
As we are saving a timestamp, we should just return an `int` and cast the stored value to `int` when returning it.